alpha_importer.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+from redis import Redis
+import config
+import requests
+import ujson
+import time
+import sys
+
+redis_connection = Redis(host = config.REDIS_HOST)
+QUEUE_KEY = 'alphaimporter:queue'
+SET_KEY_PREFIX = 'alphaimporter:set'
+ALPHA_URL = 'https://api.listenbrainz.org/'
+BETA_URL = 'http://0.0.0.0:3031'
+LISTENS_PER_GET = 100
+
+def queue_empty():
+    return redis_connection.llen(QUEUE_KEY) == 0
+
+
+def queue_front():
+    # get the first element from queue and split it to get username and auth_token
+    username, token = redis_connection.lindex(QUEUE_KEY, 0).split()
+    return username, token
+
+
+def get_batch(username, max_ts):
+    get_url = '{}/1/user/{}/listens'.format(ALPHA_URL, username)
+    payload = {"max_ts": max_ts, "count": LISTENS_PER_GET}
+    r = requests.get(get_url, params = payload)
+    if r.status_code == 200:
+        return ujson.loads(r.text)
+    else:
+        print("get from alpha http response code: {}".format(r.status_code), file = sys.stderr)
+        return None
+
+
+def get_data(batch):
+    data = {'listen_type':'import', 'payload': []}
+    for listen in batch['payload']['listens']:
+        data['payload'].append(listen)
+    return data
+
+
+def send_batch(data, token):
+    send_url = '{}/1/submit-listens'.format(BETA_URL)
+    r = requests.post(send_url, headers = {'Authorization': 'Token {}'.format(token)}, data = ujson.dumps(data))
+    if r.status_code == 200:
+        return 1
```

Comment:
Cool, I did not know of `raise_for_status`, but right now I don't know if I should use it instead of a simple if because of the logging stuff being done in the `if` block also.